kubeflow/kubeflow
# Add details to "conditions" in notebook status

resolves #2821
tested:
- create notebook from ui
- make sure notebook state transition is shown up under notebook.status.conditions

Do we need to remove notebook.status.containerStatus? (if yes, we need to update jupyter UI)

/cc @lluunn  @jlewi 


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/kubeflow/kubeflow/3319)
